product_multi_barcode_constraint_per_company/__init__.py

```diff
@@ -0,0 +1,15 @@
+from . import models
+
+from odoo.addons.product_barcode_constraint_per_company.tests.test_module import (
+    TestModule,
+)
+import unittest
+
+
+@unittest.skip
```

Comment:
to skip tests that don't work correctly without a new module

Reply:
I still don't understand why you need this. Can you please elaborate and explain why it is needed?
BTW you should not import here any test stuff.

Reply:
Without disabling the tests, there is no point in checking the functionality of the modules. 
I disabled the tests of the module product_barcode_constraint_per_company because its tests do not work correctly after we changed the module itself, that is, we moved the company_id field to a separate module, and changed the verification method. now, after we have made changes, we run the tests, they crash, the tests are due to the module product_multi_barcode. The module product_multi_barcode does not provide for the situation that one barcode can be in different companies. Because of this, we had to write a new bridge module. In which all these situations are fixed.

Reply:
@simahawk makes sense now?

Reply:
okay, I'll fix it